build.cake

```diff
@@ -92,7 +92,7 @@ Setup(context =>
 
     // Executed BEFORE the first task.
     Information("Building version {0} of NUnit.", packageVersion);
-    IsDotNetCoreInstalled = CheckIfDotNetCoreInstalled();
+    //IsDotNetCoreInstalled = CheckIfDotNetCoreInstalled();
```

Comment:
Was this checked in by mistake because you can't get the .NET Core stuff to build?